src/mainwindow/infomarkseditdlg.h

```diff
@@ -44,39 +44,41 @@ class InfoMarksEditDlg : public QDialog, private Ui::InfoMarksEditDlg
 public slots:
 	void objectListCurrentIndexChanged(const QString&);
 	void objectTypeCurrentIndexChanged(const QString&);
-	 
+  void objectClassCurrentIndexChanged(const QString&);
```

Comment:
I have no strong feelings about the convention, as long as we follow one of them. It should at least be consistent in a file. Note that L45-46 doesn't follow the Qt Code Style (if I read it correctly).

So let's settle for Qt Code Style. I'll fix this PR and the file touched by it.